2023-03-11 08:51:17,848 INFO    MainThread:8246 [wandb_setup.py:_flush():71] setting env: {'silent': 'true'}
2023-03-11 08:51:17,848 INFO    MainThread:8246 [wandb_setup.py:_flush():71] setting login settings: {}
2023-03-11 08:51:17,848 INFO    MainThread:8246 [wandb_init.py:_log_setup():371] Logging user logs to /workspace/wandb/run-20230311_085117-115zsef1/logs/debug.log
2023-03-11 08:51:17,849 INFO    MainThread:8246 [wandb_init.py:_log_setup():372] Logging internal logs to /workspace/wandb/run-20230311_085117-115zsef1/logs/debug-internal.log
2023-03-11 08:51:17,849 INFO    MainThread:8246 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f5fe5b6d210>
2023-03-11 08:51:17,851 INFO    MainThread:8246 [wandb_init.py:init():404] calling init triggers
2023-03-11 08:51:17,852 INFO    MainThread:8246 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {'CONTACT_IMG_DIR': '/workspace/input/contact_images', 'DEBUG': True, 'EXP_CATEGORY': 'DEBUG', 'EXP_NAME': 'DEBUG', 'INPUT_DIR': '/workspace/input', 'MODEL_DIR': '/workspace/output/DEBUG/model', 'OUTPUT_DIR': '/workspace/output', 'SAVED_CONTACT_CSV': '/workspace/input/Saved_contact_frames.csv', 'TRAIN_HELMET_CSV': '/workspace/input/train_baseline_helmets.csv', 'TRAIN_LABEL_CSV': '/workspace/input/train_labels.csv', 'TRAIN_TRACKING_CSV': '/workspace/input/train_player_tracking.csv', 'TRAIN_VIDEO_META_CSV': '/workspace/input/train_video_metadata.csv', 'TRAIN_VIDEO_NUM': 100, 'T_max': 10, 'VALID_VIDEO_NUM': 10, 'batch_size': 32, 'features': ['x_position_1', 'y_position_1', 'x_position_2', 'y_position_2', 'speed_1', 'distance_1', 'direction_1', 'orientation_1', 'acceleration_1', 'sa_1', 'speed_2', 'distance_2', 'direction_2', 'orientation_2', 'acceleration_2', 'sa_2', 'players_dis'], 'img_size': [224, 224], 'inp_channels': 3, 'kaggle': False, 'lr': 0.0001, 'masksize_helmet_ratio': 4, 'min_lr': 1e-08, 'model_name': 'swin_s3_tiny_224', 'n_epoch': 2, 'n_folds': 3, 'num_img_feature': 10, 'num_workers': 2, 'out_features': 20, 'pretrained': True, 'print_freq': 1000, 'random_seed': 21, 'roll_sum_window_size': 10, 'sample_num': 1000, 'train_folds': [0, 1, 2], 'wandb': True, 'weight_decay': 1e-06}
2023-03-11 08:51:17,852 INFO    MainThread:8246 [wandb_init.py:init():460] starting backend
2023-03-11 08:51:17,853 INFO    MainThread:8246 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-03-11 08:51:17,869 INFO    MainThread:8246 [backend.py:ensure_launched():216] starting backend process...
2023-03-11 08:51:17,878 INFO    MainThread:8246 [backend.py:ensure_launched():222] started backend process with pid: 8377
2023-03-11 08:51:17,879 INFO    MainThread:8246 [wandb_init.py:init():469] backend started and connected
2023-03-11 08:51:17,892 INFO    MainThread:8246 [wandb_run.py:_label_probe_notebook():911] probe notebook
2023-03-11 08:51:17,893 ERROR   MainThread:8246 [jupyter.py:attempt_kaggle_load_ipynb():279] Unable to load kaggle notebook
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/wandb/jupyter.py", line 273, in attempt_kaggle_load_ipynb
    client = kaggle.UserSessionClient()
  File "/root/.local/lib/python3.7/site-packages/kaggle_session.py", line 14, in __init__
    self.web_client = KaggleWebClient()
  File "/root/.local/lib/python3.7/site-packages/kaggle_web_client.py", line 30, in __init__
    'A JWT Token is required to call Kaggle, '
kaggle_web_client.CredentialError: A JWT Token is required to call Kaggle, but none found in environment variable KAGGLE_USER_SECRETS_TOKEN
2023-03-11 08:51:17,894 INFO    MainThread:8246 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-11 08:51:17,894 INFO    MainThread:8246 [wandb_init.py:init():533] updated telemetry
2023-03-11 08:51:17,964 INFO    MainThread:8246 [wandb_init.py:init():563] communicating current version
2023-03-11 08:51:18,796 INFO    MainThread:8246 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.13.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-11 08:51:18,796 INFO    MainThread:8246 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2023-03-11 08:51:19,095 INFO    MainThread:8246 [wandb_init.py:init():606] starting run threads in backend
2023-03-11 08:51:24,096 INFO    MainThread:8246 [wandb_run.py:_console_start():1810] atexit reg
2023-03-11 08:51:24,098 INFO    MainThread:8246 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2023-03-11 08:51:24,099 INFO    MainThread:8246 [wandb_run.py:_redirect():1721] Wrapping output streams.
2023-03-11 08:51:24,100 INFO    MainThread:8246 [wandb_run.py:_redirect():1745] Redirects installed.
2023-03-11 08:51:24,100 INFO    MainThread:8246 [wandb_init.py:init():633] run started, returning control to user process
2023-03-11 08:51:24,100 INFO    MainThread:8246 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:51:24,101 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:51:24,229 INFO    MainThread:8246 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:51:24,231 INFO    MainThread:8246 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:51:24,232 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:51:24,315 INFO    MainThread:8246 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:51:25,933 INFO    MainThread:8246 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:51:25,934 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:51:26,034 INFO    MainThread:8246 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:51:31,458 INFO    MainThread:8246 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:51:31,458 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:51:31,601 INFO    MainThread:8246 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:51:31,743 INFO    MainThread:8246 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:51:31,744 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:51:31,808 INFO    MainThread:8246 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:51:34,503 INFO    MainThread:8246 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:51:34,503 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:51:34,648 INFO    MainThread:8246 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:51:36,786 INFO    MainThread:8246 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:51:36,787 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:51:36,952 INFO    MainThread:8246 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:51:36,968 INFO    MainThread:8246 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:51:36,968 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:51:37,046 INFO    MainThread:8246 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:51:43,358 INFO    MainThread:8246 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:51:43,358 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:52:23,673 INFO    MainThread:8246 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:52:23,676 INFO    MainThread:8246 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:52:23,677 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:52:23,780 INFO    MainThread:8246 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:52:23,783 INFO    MainThread:8246 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:52:23,784 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:52:23,866 INFO    MainThread:8246 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:52:23,873 INFO    MainThread:8246 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:52:23,873 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:52:24,009 INFO    MainThread:8246 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:52:24,012 INFO    MainThread:8246 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:52:24,012 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:52:24,146 INFO    MainThread:8246 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:52:24,205 INFO    MainThread:8246 [wandb_setup.py:_flush():71] setting env: {'silent': 'true'}
2023-03-11 08:52:24,206 INFO    MainThread:8246 [wandb_setup.py:_flush():71] setting login settings: {}
2023-03-11 08:52:24,207 INFO    MainThread:8246 [wandb_setup.py:_flush():71] [34m[1mwandb[0m: wandb version 0.13.11 is available!  To upgrade, please run:
[34m[1mwandb[0m:  $ pip install wandb --upgrade
2023-03-11 08:52:24,208 INFO    MainThread:8246 [wandb_init.py:_log_setup():371] Logging user logs to /workspace/wandb/run-20230311_085224-hd60o0g1/logs/debug.log
2023-03-11 08:52:24,209 INFO    MainThread:8246 [wandb_init.py:_log_setup():372] Logging internal logs to /workspace/wandb/run-20230311_085224-hd60o0g1/logs/debug-internal.log
2023-03-11 08:52:24,212 INFO    MainThread:8246 [wandb_init.py:init():404] calling init triggers
2023-03-11 08:52:24,213 INFO    MainThread:8246 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {'CONTACT_IMG_DIR': '/workspace/input/contact_images', 'DEBUG': True, 'EXP_CATEGORY': 'DEBUG', 'EXP_NAME': 'DEBUG', 'INPUT_DIR': '/workspace/input', 'MODEL_DIR': '/workspace/output/DEBUG/model', 'OUTPUT_DIR': '/workspace/output', 'SAVED_CONTACT_CSV': '/workspace/input/Saved_contact_frames.csv', 'TRAIN_HELMET_CSV': '/workspace/input/train_baseline_helmets.csv', 'TRAIN_LABEL_CSV': '/workspace/input/train_labels.csv', 'TRAIN_TRACKING_CSV': '/workspace/input/train_player_tracking.csv', 'TRAIN_VIDEO_META_CSV': '/workspace/input/train_video_metadata.csv', 'TRAIN_VIDEO_NUM': 100, 'T_max': 10, 'VALID_VIDEO_NUM': 10, 'batch_size': 32, 'features': ['x_position_1', 'y_position_1', 'x_position_2', 'y_position_2', 'speed_1', 'distance_1', 'direction_1', 'orientation_1', 'acceleration_1', 'sa_1', 'speed_2', 'distance_2', 'direction_2', 'orientation_2', 'acceleration_2', 'sa_2', 'players_dis'], 'img_size': [224, 224], 'inp_channels': 3, 'kaggle': False, 'lr': 0.0001, 'masksize_helmet_ratio': 4, 'min_lr': 1e-08, 'model_name': 'swin_s3_tiny_224', 'n_epoch': 2, 'n_folds': 3, 'num_img_feature': 10, 'num_workers': 2, 'out_features': 20, 'pretrained': True, 'print_freq': 1000, 'random_seed': 21, 'roll_sum_window_size': 10, 'sample_num': 1000, 'train_folds': [0, 1, 2], 'wandb': True, 'weight_decay': 1e-06}
2023-03-11 08:52:24,213 INFO    MainThread:8246 [wandb_init.py:init():426] re-initializing run, found existing run on stack: 115zsef1
2023-03-11 08:52:24,214 INFO    MainThread:8246 [wandb_run.py:finish():1457] finishing run stgkrt/NFL/115zsef1
2023-03-11 08:52:24,215 INFO    MainThread:8246 [jupyter.py:save_history():429] not saving jupyter history
2023-03-11 08:52:24,216 INFO    MainThread:8246 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:52:24,217 INFO    MainThread:8246 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2023-03-11 08:52:24,218 INFO    MainThread:8246 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2023-03-11 08:52:24,220 INFO    MainThread:8246 [wandb_run.py:_restore():1752] restore
2023-03-11 08:52:24,802 INFO    MainThread:8246 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 873
  total_bytes: 873
}

